spec/langchain/agent/sql_query_agent/sql_query_agent_spec.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 RSpec.describe Langchain::Agent::SQLQueryAgent do
-  let(:db) { Langchain::Tool::Database.new(connection_string: "mock:///") }
+  let(:db) { Langchain::Tool::Database.new(connection_string: "mock://postgres") }
```

Comment:
Note that `mock:///` can't be used here, since that class doesn't implement all methods (like `table`).